### PR TITLE
#47: Named the return column as predictions

### LINKED ISSUE #47: Name the return column as predictions
- prediction return columns is named as "predictions"
- but scania trucks classification  return predictions in  the target column, rather than "predictions" columns

### PR DESCRIPTION
**Pull request by umitbuyuksahin**
Fixes #47

### CODE REVIEW

File: tests/integration_tests/test_autopilot_prediction_udf_real.py [hunk 1]
```diff
@@ -42,6 +42,7 @@ def get_connection(self, name: str) -> Connection:
 class Context:
     def __init__(self):
         self._emitted = []
+        self._is_emitted_once = False
```
Comment (tomuben) [inline tests/integration_tests/test_autopilot_prediction_udf_real.py:45]: I am not sure, but maybe the name is a bit confusing here.
Maybe better:
```suggestion
        self._is_accessed_once = False
```

**Review by tomuben — DISMISSED** [review]

**Review by tomuben — COMMENTED** [review]
Also don't forget to update the changelog.

File: tests/integration_tests/test_autopilot_prediction_udf_real.py [hunk 2]
```diff
@@ -55,7 +56,9 @@ def get_dataframe(self, num_rows='all'):
             (2.2, 2.4),
             (3.3, 3.6)
         ]
-        return pd.DataFrame(data=data)
+        return_df = None if self._is_accessed_once else pd.DataFrame(data=data)
+        self._is_emitted_once = True
```
Comment (tomuben) [inline tests/integration_tests/test_autopilot_prediction_udf_real.py:60]: Need to change name here, too

Comment (umitbuyuksahin) [inline tests/integration_tests/test_autopilot_prediction_udf_real.py:60]:
```suggestion
        self._is_accessed_once = True
```

**Review by tomuben — APPROVED** [review]

**Pull request merged**